up-l1/zenoh.adoc

```diff
@@ -178,3 +183,91 @@ Encoding::WithSuffix(
 ----
 
 * Zenoh takes data as a buffer simply and sends it directly. Ignore the shared memory (reference) temporarily.
+
+=== Work Flow
+
+The section provides the details how it works in different message types.
+
+==== Transfer Publish message
+
+1. A uEntity (`origin-authority/origin-entity`) creates a message with
+    * source: `up://origin-authority/origin-entity/origin-version/origin-resource.instance#message`
+    * sink: `-`
+
+2. `UTransport.send` maps source to Zenoh key
+    * local (w/o origin-authority): `upl/` + `origin-resource-id + origin-entity-id + origin-version`
+    * remote (with origin-authority): `upr/` + `origin-authority-id/` + `origin-resource-id + origin-entity-id + origin-version`
+
+3. Subscriber registers a listener for
+    * `up://origin-authority/origin-entity/origin-version/origin-resource.instance#message`
+
+4. which `UTransport.registerListener` maps to Zenoh key
+    * local (w/o origin-authority): `upl/` + `origin-resource-id + origin-entity-id + origin-version`
+    * remote (with origin-authority): `upr/` + `origin-authority-id/` + `origin-resource-id + origin-entity-id + origin-version`
+
+5. Since this is **Publish URI**, `Zenoh subscriber` will be used to bind the `registerListener`
```

Comment:
For the section on `registerListener` for Publish, Notification, Request, and Response I think it'd be a good idea to explicitly make note of which UUri we're registering on: destination or origin.

Reply:
I agree with you. It's clearer to readers.